scrapy/mail.py

```diff
@@ -68,10 +60,6 @@ def send(self, to, subject, body, cc=None, attachs=(), mimetype='text/plain', ch
         msg['To'] = COMMASPACE.join(to)
         msg['Date'] = formatdate(localtime=True)
         msg['Subject'] = subject
-        rcpts = to[:]
-        if cc:
-            rcpts.extend(cc)
-            msg['Cc'] = COMMASPACE.join(cc)
```

Comment:
after moving it out of here `cc` argument seems to be unused

Reply:
`cc` is being used again in `create_email_message` function.